code/part-two/client/.babelrc

```diff
@@ -1,3 +1,17 @@
 {
-  "presets": ["env", "react"]
-}
\ No newline at end of file
+  "presets": [
+    [
+      "env",
+      {
+        "targets": {
+          "browsers": [
+            "> 5%",
```

Comment:
Wouldn't increasing this number reduce the number of browsers you are supporting, and therefore reduce the amount of transpiling your are doing? Async/await only worked when you reduced how much you transpiled?

Reply:
Yes, it will reduce the number of browsers that are being supported and reduce the amount of transpiling that's happening. Yes, I could only get `async/await` to work when I reduced how much I transpiled bc I ran into an error in the browser otherwise. My initial research led me to the solution in this PR, but now that I'm thinking more about this courtesy of your PR review, I'm now wondering: "does `async/await` transpilation support require an extra plugin?"

which (a) would be kinda dumb imo bc isn't that what `babel-preset-env` is supposed to handle? and (b) would be a better solution such that we can support more browsers!

I will create a new issue and see how I can tackle this in a future PR.

Reply:
Tracking this at https://github.com/hyperledger/education-cryptomoji/issues/118 now